src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -28,7 +28,87 @@ public ContributionProvider(Docset docset, GitHubUserCache gitHubUserCache, GitC
             LoadCommitsTime(docset);
         }
 
-        public async Task<(List<Error> error, Contributor author, List<Contributor> contributors, DateTime updatedAt)> GetAuthorAndContributors(
+        public async Task<(List<Error> errors, ContributionInfo contributionInfo)> GetContributionInfo(Document document, SourceInfo<string> authorName)
+        {
+            var (errors, author, contributors, updatedAt) = await GetAuthorAndContributors(document, authorName);
+
+            return (errors, updatedAt != default
```

Comment:
nit: I would directly rename `GetAuthorAndContributors` to `GetContributionInfo` and make it return `(Errors, ContributionInfo)`, there is no need to create a wrapper function just to convert the return value type.